modules/ROOT/pages/ai-proxy.adoc

```diff
@@ -0,0 +1,91 @@
+= AI proxy server reference guide
+:navtitle: AI proxy server reference guide
+:description: A guide to adding a proxy server to a TinyMCE instance running the AI Assistant plugin.
+:description_short: Running a proxy server and the AI Assistant plugin.
+:keywords: plugin, ai, assistant, proxy server
+:pluginname: AI Assistant
+:plugincode: ai
+:plugincategory: premium
+
+include::partial$misc/admon-ai-pricing.adoc[]
+
+
+== What is an AI proxy server?
+
+A proxy is a server that sits between the browser containing the {productname} editor and the AI Large Language Model (LLM); for example, the OpenAI server.
+
+With a proxy server in place, {productname} does not communicate directly with the AI LLM. Instead requests go through the proxy.
+
+The LLM’s responses, in return, also go through the proxy and then to {productname}.
+
+The proxy adds a layer of flexibility by allowing extra processing before the request is sent to the LLM and before returning the response to the editing session in {productname}.
+
+== Why use a proxy service?
+
+* To hide the OpenAI API key used for queries.
+  - This is https://help.openai.com/en/articles/5112595-best-practices-for-api-key-safety[recommended by OpenAI].
+* It allows multiple queries from different {productname} sessions to work with one OpenAI key.
+* It can, optionally, validate logged-in users and reject unauthorised usage of the OpenAI service.
```

Comment:
Should this be `unauthorized` rather than `unauthorised` here?

```suggestion
* It can, optionally, validate logged-in users and reject unauthorized usage of the OpenAI service.
```